test/data/ross-kriel-purdon.yml

```diff
@@ -0,0 +1,18 @@
+---
+:class: MemberPage
+:url: https://www.pa.org.za/person/ross-kriel-purdon/
+:to_h:
+  :id: ross-kriel-purdon
+  :name: Ross Kriel Purdon
+  :honorific_prefix: ''
+  :party: Democratic Alliance
+  :party_id: DA
+  :party_wikidata_id: Q761877
+  :area: National
+  :area_wikidata_id:
+  :email: rpurdon2010@gmail.co
```

Comment:
BTW: That seems unlikely as email address. It's what the site says, so it's not strictly _wrong_ to use it here, but I'd have a strong suspicion that this is a data entry problem, and baking it into a test doesn't seem ideal.